cmake/Findamxc.cmake

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+# SPDX-FileCopyrightText: 2019-2020 the prplMesh contributors (see AUTHORS.md)
```

Comment:
```suggestion
# SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
```

Reply:
Thank you for paying attention! Applied. Resolving.